packages/applicant/src/pages/index.page.tsx

```diff
@@ -40,7 +40,7 @@ function HomePage({ loginUrl, registerUrl, oneLoginEnabled }: HomePageProps) {
               funding organisations make a decision about who to award funding
               to.
             </p>
-            {oneLoginEnabled == 'true' ? (
+            {oneLoginEnabled === 'true' ? (
```

Comment:
Are we absolutely sure oneLoginEnabled is interpreted as a string 👀

Reply:
env vars in node.js are always strings afaik. Here's some proof for this case
![image](https://github.com/cabinetoffice/gap-find-apply-web/assets/135222889/2af6300c-6b3f-49b4-8dc8-3a5ee2145794)